less/home-page.less

```diff
@@ -1,61 +1,142 @@
+.content-header {
+  width: 100%;
+  border-bottom: 2px dashed silver;
+
+  h1 {
+    margin-top: 2rem;
+    margin-bottom: -2rem;
+    margin-left: 55rem;
+    color: @font-color;
+
+    @media @mobile {
+      margin-left: 18rem;
+      margin-bottom: 2rem;
+      font-size: 3.5rem;
+    }
+  }
+
+}
+
```

Comment:
![image](https://user-images.githubusercontent.com/8473052/57963585-10c1cb00-78f4-11e9-8c99-30a3ff82da76.png)

Oof. So I'd recommend using flexbox here to space out these elements! I removed a lot of the margins and added a few properties to the parent element and this seems to flow a bit better:

![image](https://user-images.githubusercontent.com/8473052/57963601-58485700-78f4-11e9-92ad-1df50ced3b33.png)

And for the mobile design, you'd need to use a column `flex-direction`.